client/src/components/Header.js

```diff
@@ -56,6 +57,39 @@ function Header() {
     setAnchorEl(null);
   };
 
+  const logout = () => {
+    let status
+    fetch("/logout", {
+      method: "POST",
+      credentials: "include"
+    })
+    .then(res => {
+        status = res.status
+
+        if (status < 500) return res.json();
+        else throw Error("Server error");
+    })
+    .then(res=> {
+        if (status === 200) {
+          alert(res.response)
+          setAnchorEl(null)
+          history.push("/login")
+        }
+        else if (status === 400) {
+          alert(res.response)
+          setAnchorEl(null)
+        }
+        else 
+          throw Error("Server error");
```

Comment:
@bugatti12345 Added throw error that takes into account of other 4xx/5xx status errors